src/components/Tasks/Task/index.js

```diff
@@ -0,0 +1,46 @@
+import React, { useRef, useContext } from "react";
```

Comment:
You should use single quotes in js files to match the rest of the project.

Reply:
Sorry, I'm updated all changed files to get back single quotes, and I add .editorconfig.